install_xapian.sh

```diff
@@ -8,6 +8,11 @@ if [ -z "$VERSION" ]; then
     exit 1
 fi
 
+JFLAG=1
+if [ "$2" = "--use-all-cores" ]; then
+  JFLAG=$(($(getconf _NPROCESSORS_ONLN) + 1))
+fi
```

Comment:
This seems to work on current macOS and FreeBSD, though some older FreeBSD may only have `NPROCESSORS_ONLN` (no leading underscore). That seems to fail in a way that limits us to 1 processor and then proceeds, so that's probably fine.